src/ucp/rma/rma_send.c

```diff
@@ -148,6 +153,13 @@ ucp_rma_request_init(ucp_request_t *req, ucp_ep_h ep, const void *buffer,
         return UCS_OK;
     }
 
+    status = ucp_send_request_set_user_memh(req,
+                                            ucp_ep_config(ep)->key.rma_md_map,
```

Comment:
configuration key should contain only direct result of wireup
so rma_md_map should be in ucp_ep_config_t - not the key - and set during ucp_ep_config_init

Reply:
I followed the same idea as we have for `rma_bw_md_map`.
Shall I move both?

Reply:
ok, let's keep as-is, will probably be able to remove it soon anyway after using ucp rcache for proto v1 (since then md_map will not be needed for ucp_send_request_set_user_memh )